docs/development/getting_started.rst

```diff
@@ -150,12 +146,29 @@ different version, the path to ``virtualenvwrapper.sh`` will differ. Running
 .. _Homebrew: https://brew.sh/
 .. _Docker: https://store.docker.com/editions/community/docker-ce-desktop-mac
 
-Clone the repository
---------------------
+Fork & Clone the repository
+---------------------------
+
+Now you are ready to get your own copy of the source code.
+Visit our repository_ fork it and clone it on you local machine:
+
+
+.. code:: sh
+
+   git clone git@github.com:<your_github_username>/securedrop.git
+
+.. _repository: https://github.com/freedomofpress/securedrop
+
+Install python requirments
+--------------------------
 
-Once you've installed the prerequisites for the development environment,
-use git to clone the SecureDrop repo.
+SecureDrop uses many 3rd party open source packages from the python community.
+Ensure your virtualenv is activated and install the packages.
 
 .. code:: sh
 
-   git clone https://github.com/freedomofpress/securedrop.git
+    pip install -r securedrop/requirements/develop-requirements.txt
+    pip install -r testinfra/requirements.txt
+
+You will need to run this everytime new packages are added so if you get
+an ``ImportError`` run theose two lines again.
```

Comment:
The warning above could be:
<pre>
.. note:: You will need to run this everytime new packages are added.
</pre>
I think the 
<pre>
so if you get an ``ImportError`` run theose two lines again.
</pre>
can be left out.